docs/scripts/dc-restic-post-restore.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+OADP_NAMESPACE=${OADP_NAMESPACE:=openshift-adp}
+
+if [[ $# -ne 1 ]]; then
+  echo "usage: ${BASH_SOURCE} restore-name"
+  exit 1
+fi
+
+echo using OADP Namespace $OADP_NAMESPACE
+echo restore $1
+
+echo Deleting disconnected restore pods
+oc delete pods -l openshift.io/disconnected-from-dc=$1
```

Comment:
Per https://github.com/openshift/openshift-velero-plugin/pull/141/files#r891349632 please update all new labels to `oadp.openshift.io`

Reply:
Updated.